js/build.js

```diff
@@ -191,25 +197,23 @@ if (typeof jQuery !== 'undefined') {
     $(this).addClass('sending');
     holder.addClass('sending');
 
-    $message.val('');
```

Comment:
watch out, because this means if I press twice the button it will send the message twice